target/linux/pistachio/config-5.15

```diff
@@ -1,33 +1,39 @@
 CONFIG_ARCH_32BIT_OFF_T=y
 CONFIG_ARCH_HIBERNATION_POSSIBLE=y
+CONFIG_ARCH_KEEP_MEMBLOCK=y
 CONFIG_ARCH_MMAP_RND_BITS_MAX=15
 CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MAX=15
 CONFIG_ARCH_SUSPEND_POSSIBLE=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
-CONFIG_BLK_SCSI_REQUEST=y
+# CONFIG_BOARD_INGENIC is not set
 CONFIG_BOARD_SCACHE=y
-CONFIG_BOOT_ELF32=y
+CONFIG_BUILTIN_DTB=y
```

Comment:
Why do you activate `CONFIG_BUILTIN_DTB` ? I think the dtb is in a separate file on the sd card.

Reply:
Was automatically activated by make kernel_oldconfig. I looked at some other targets and `malta` and `octeon` have it also activated.